bigbluebutton-html5/imports/ui/components/modal/base/styles.scss

```diff
@@ -22,9 +22,7 @@
 
 .overlay {
   z-index: 1000;
-  // background: transparentize($color-white, .35);
-  background: #fff;
-  // background: transparentize($color-gray-dark, .40);
+  background: transparentize($color-gray-dark, .40);
```

Comment:
We want a more compact Audio modal because the way it is now it blocks the entire screen when you load the client. You can't see the meeting name, users, or any other context of the meeting. You basically see one big modal. We need to be able to see some of the context for the meeting and a way to see it is via making this particular modal more compact